prac_07/myguitars.py

```diff
@@ -6,12 +6,14 @@
 
 from guitar import Guitar
 
+FILE_NAME = 'guitars.csv'
+
 
 def main():
```

Comment:
should you add error handling for the reading of the file

Reply:
good point